### ISSUE TITLE
ValueError: <table_name>: Column <column_name> is entirely NULL!

### ISSUE THREAD

**Issue by jdangerx** [catalyst-cooperative/pudl#2877]
```[tasklist]
### Assets
- [ ] plants_hydro_ferc1
- [ ] electricity_sales_by_rate_schedule_ferc1

```

**Comment by e-belfer** [comment]
I'm not able to reproduce this issue on my branch #2881.

**Comment by jdangerx** [comment]
Fixed by #2883.
